src/big/big_decimal.cr

```diff
@@ -114,22 +114,36 @@ struct BigDecimal
     scale = @scale - other.scale
     n, d = @value, other.@value
 
-    quotient, remainder = n.tdiv(d), n.remainder(d)
-    return BigDecimal.new(quotient, scale) if remainder == ZERO
+    # quotient, remainder = n.tdiv(d), n.remainder(d)
+    quotient, remainder = n.divmod(d)
+    puts "n #{n} d #{d} q #{quotient} r #{remainder}"
```

Comment:
Debug leftovers.

Reply:
Yeah, intermediate commits to save work. Removed afterwards